packages/web-components/fast-foundation/src/toolbar/toolbar.ts

```diff
@@ -183,12 +174,12 @@ export class Toolbar extends FoundationElement {
             return true;
         }
 
-        const incrementer = this.getDirectionalIncrementer(key);
-        if (!incrementer) {
+        const incrementor = this.getDirectionalIncrementor(key);
+        if (!incrementor) {
             return !(e.target as HTMLElement).closest("[role=radiogroup]");
         }
 
-        const nextIndex = this.activeIndex + incrementer;
+        const nextIndex = this.activeIndex + incrementor;
```

Comment:
I can't find any concrete difference in usage for `incrementer` or `incrementor`, and most sources just point to the Wiktionary entries:

[Incrementer](https://en.wiktionary.org/wiki/incrementer):
> * Any device or circuit that carries out an incrementation operation.
> * _Related terms: incrementor_

[Incrementor](https://en.wiktionary.org/wiki/incrementor):
> * That which increments; a computer programming instruction that increments a value.
> * _Related terms: incrementer_

🤷🏻

Reply:
ahah ahh yes, i just was using my spellcheck, i'l leave this change out.